src/renderer/hooks/useKeyImport.ts

```diff
@@ -13,11 +13,82 @@ export default function useKeyImport() {
   const [key, setKey] = useState<ImportData>({});
   const [error, setError] = useState<string | null>(null);
 
+  const isValidFileName = (fileName: string) => {
+    const utcRegex =
+      /^UTC--\d{4}-\d{2}-\d{2}T\d{2}-\d{2}-\d{2}Z--[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}.json$/i;
+    return utcRegex.test(fileName);
```

Comment:
```suggestion
    return /^(?:UTC--([0-9]{4}-[0-9]{2}-[0-9]{2})T([0-9]{2}-[0-9]{2}-[0-9]{2})Z--)?([0-9a-f]{8}-(?:[0-9a-f]{4}-){3}[0-9a-f]{12})(?:.json)?$/i.test(
      fileName,
    );
```
Regex가 유효하지 않아서 조금 수정하고, const 선언 없이 바로 리턴되도록 했습니다.